frontend/src/metabase/entities/users.js

```diff
@@ -21,6 +21,31 @@ export const PASSWORD_RESET_MANUAL =
   "metabase/entities/users/RESET_PASSWORD_MANUAL";
 export const RESEND_INVITE = "metabase/entities/users/RESEND_INVITE";
 
+const BASE_FORM_FIELDS = [
+  // $FlowFixMe - complaining about title not being in object literal?
```

Comment:
typing like

```
import type { FormFieldDef } from "metabase/containers/Form";

const BASE_FORM_FIELDS: FormFieldDef[] = [
```

should do it.

You'll need to

```
export type FormFieldDef
```

in `metabase/containers/Form.jsx`